### PR TITLE
Add support for Tau board

### PR DESCRIPTION
**Pull request by ealasu**

### CODE REVIEW

**Comment by dhalbert** [comment]
@ladyada Could you opine on the VID/PID? It's the Arduino VID. Otherwise looks fine.

**Review by dhalbert — CHANGES_REQUESTED** [review]
The VID belongs to Arduino. Please get a PID from https://pid.codes/ or if you are volume producer, consider buying your own VID.

**Comment by ladyada** [comment]
@scswift are you still selling RabidPrototypes hardware?

**Comment by scswift** [comment]
Yes, I still sell the Tau boards, but once my remaining stock is depleted I won't be manufacturing more. I've only got 100 of them left.

**Comment by ladyada** [comment]
@scswift ok, did you get a USB VID/PID for your products?

**Comment by scswift** [comment]
I have not. The Tau is the only product I have which required a unique PID because they use the SAMD21E17A. But they're nearly sold out, so there's little point in updating them now, and if I were to do so, everyone who's purchased them over the last four years and tries to download the latest drivers would find they could not access their hardware, which would result in major headaches all around. So I'd rather you just opt not to support the board if you're concerned about driver conflicts.

**Comment by ladyada** [comment]
@scswift kk where did the unique PID come from?

**Comment by scswift** [comment]
I don't recall how it was selected.

**Comment by ladyada** [comment]
@scswift ok, for the rest of your products, do you recall how those USB PIDs were selected? 
https://rabidprototypes.com/product-category/arduino/
they are all samd chips... did you 'borrow' an arduino vid/pid for those?

**Comment by scswift** [comment]
The Tau (100 in stock, end of life), Firecricket (7 left in stock, EOL), and Pixel Mini (out of stock forever), use the same chip + pinout and same VID/PID, and the PID is unique out of necessity. 

The Neutrino (3 left in stock, EOL), Firecricket XL (20 remaining in stock, EOL), and Pixel (out of stock forever) are chip + pin compatible with the Arduino Zero, and so used one of its VID/PID combos, because otherwise their drivers would always be out of date as I couldn't possibly keep up with the constant updates to the IDE.

The Tachyon never went into production so no VID / PID was selected for it, but if it ever does, I'll be sure to select unique ones for it using that website you linked.

The Positron uses the Atmega1284P, isn't a general purpose board, and doesn't use USB.

Basically, the general-purpose Arduino side of my business is on its last legs and I'm almost out of stock on everything. The boards never sold very well, so I'm transitioning back to making purpose-built electronics for cosplay props which have a higher profit margin.

**Comment by DirtyEngineer** [comment]
> The Tachyon never went into production so no VID / PID was selected for it, but if it ever does,

You really owe your 229 Kickstarter backers an update concerning this. It's been 2 and a half years since you launched this campaign and you still haven't delivered.  Issue refunds to everyone and give us closer. https://www.kickstarter.com/projects/rabidprototypes/tachyon-the-lightning-fast-120mhz-arduino-compatib/description

**Comment by scswift** [comment]
I have been providing updates. And I've refunded everyone who's asked, including you.

**Comment by DirtyEngineer** [comment]
> I have been providing updates. And I've refunded everyone who's asked, including you.
The only update posted is from August 29, 2018.  Your last comment posted is from 4 months ago.
Let's be blunt here, you admitted that you have avoided posting an actual update as it will be sent to all backers and you don't want to deal with the repercussions of reminding people that they backed your campaign. You only "update" via the comments section and offer refunds to people who ask which I have never done. You issued me a refund because I've called you to task.

Are you hoping that the majority of people forget about backing your campaign and you can keep their money?

**Comment by ladyada** [comment]
hi folks - this thread isn't a place to hash out kickstarters. we're just trying to figure out whether there was ever an official VID / PID allocated to this board. it seems not! so this topic is now locked
@dhalbert please allocate one from the adafruit pool for this PR so we are not using Arduino's VID

**Comment by dhalbert** [comment]
Hi @scswift @ealasu we have allocated:

Rabid Prototypes Tau bootloader 0x239A  0x00E9
Rabid Prototypes Tau arduino 0x239A 0x80E9

I will update this PR.

**Review by ladyada — APPROVED** [review]
